README.md

```diff
@@ -77,32 +71,46 @@ The idea was to generate a number of random pieces of music split into different
 
 Feel free to download and play any of the `Sample.mid` files to see Johann's work in action!
 
-
 ### Built With
 
-* [Python](https://www.python.org/)
+- [Python](https://www.python.org/)
 
 This was built purely in Python - we used a couple small packages in order to play the music and generate MIDI files, but that's all that was used.
 
+### How To Run
```

Comment:
Would you mind adding the "How To Run" section to the table of contents at the top of the file?